packages/.coderabbit.yaml

```
@@ -0,0 +1,196 @@
+                - Avoid `any`; use `unknown` when necessary.
+                - Use runtime type-checking for environment variables (e.g., Zod).
+                - Prevent circular dependencies with the internal module pattern.
+                - Libraries should have an `external.ts` file explicitly listing public exports.
+                - Use `bigint` as-is; cast to `Number` only when exposing values via APIs.
+                - Document all code with JSDoc.
+                - Encourage static async factory functions for constructors.
+        - Avoid overly nitpicky feedback beyond these best practices.
+        - path: "**/*.tsx"
+          instructions: |2-
+          Review `.tsx` files with these front-end best practices:
+                - Avoid complex logic in the render section; abstract it into functions.
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 14 but found 16

(indentation)